FEATURES.md

```diff
@@ -266,28 +267,35 @@ Floating point numbers must have the decimal point `.` and can optionally take a
 The three number types are fundamentally different.
 If you mix two numbers of different types HVM will interpret the binary representation of one of them incorrectly, leading to incorrect results. Which number is interpreted incorrectly depends on the situation and shouldn't be relied on for now.
 
-At the moment Bend doesn't have a way to convert between the different number types, but it will be added in the future.
-
-You can use `switch` to pattern match on unsigned native numbers:
+Bend now has a way to convert between the different number types!
```

Comment:
This is strangely worded and doesn't fir the rest of the file.
Something like "You can convert between the number types using the builtin casting functions" would be better i think